mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return price * 1.2;
```

Comment:
We just want the tax, not the price + the tax.
So to get 20% of the price, you need to multiply by a number less than one